src/main/java/project/bot/service/MessageSender.java

```diff
@@ -0,0 +1,83 @@
+package project.bot.service;
+
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+
+/**
+ * @author Vladyslav Pustovalov
+ * class which consist of answers to a user from the Telegram Bot
+ */
+public class MessageSender {
```

Comment:
you can shorten methods to rebese logic from them to private method foo(long id, string text)